test/BuildingRegistry.Tests/BackOffice/TestBackOfficeContext.cs

```diff
@@ -0,0 +1,26 @@
+namespace BuildingRegistry.Tests.BackOffice
+{
+    using System;
+    using BuildingRegistry.Api.BackOffice.Abstractions;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Design;
+
+    public class TestBackOfficeContext : BackOfficeContext
```

Comment:
`TestBackOfficeContext` Or `FakeBackOfficeContext`?
We use fake in other registries, but if you choose test, rename the factory.